### PR TITLE
Uks translations

### PR DESCRIPTION
**Pull request by mruks**
Dutch and French additions

### CODE REVIEW

**Comment by mruks** [comment]
Finishing to add Dutch and French

**Comment by mruks** [comment]
I thought by closing that it will be added, lol

**Comment by mruks** [comment]
Miss clicked, officially reopening the request

File: en-GB/Settings.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default {
     },
     'RevealEggs': {
       'Title': 'Reveal Eggs',
-      'Description': 'Should an eggs contents be revealed in the party screen'
+      'Description': 'Should an egg\'s contents be revealed in the party screen'
```
Comment (xLink) [inline en-GB/Settings.js]: Should an egg is contents be revealed? doesnt make sense

**Pull request merged**